requirements.txt

```diff
@@ -1,4 +1,4 @@
 Flask
 Flask-PyMongo
 Flask-WTF
--e git://github.com/in-toto/in-toto.git@v0.2.3.dev5#egg=in-totov0.2.3.dev5
+in-toto
```

Comment:
I'm against un-pinning in-toto:
See https://github.com/in-toto/layout-web-tool/pull/37#discussion_r450067804 and https://www.promptworks.com/blog/pin-all-dependencies.

Reply:
Very interesting, and I'd missed the previous comment, my bad.

Should we pin all of them to their latest versions in this PR?

Reply:
Yes, it's probably a good idea to pin at least major version of the others too.